source/Integrate/Tutorials/analytics_with_keen_io.md

```diff
@@ -15,145 +15,196 @@ It only takes a few minutes to start streaming data over and once you do you can
-- [Analyzing and Visualizing your SendGrid Data](#-analyzing-your-sendgrid-data)
-- [Doing Drilldowns & Reporting on Your Data](#-doing-drilldowns-and-reporting-on-your-data)
-- [Embedding SendGrid Analytics Into Your App](#-embedding-sendgrid-analytics-into-your-app)
-
-{% anchor h3 %}
-Sending your SendGrid Data to Keen IO
+- [Send Your SendGrid Data to Keen IO](#send-your-sendgrid-data-to-keen-io)
+- [Test Your Integration](#test-your-integration)
+- [SendGrid Email Analytics App: How to Guide](#sendgrid-email-analytics-how-to-guide)
+- [Ad-Hoc Querying & Raw Data Access](#ad-hoc-querying-and-raw-data-access)
+- [Embedding SendGrid Analytics Into Your App](#embedding-sendgrid-analytics-into-your-app)
+- [Additional Resources](#additional-resources)
+
+{% anchor h3 send-your-sendgrid-data-to-keen-io %}
+Send your SendGrid Data to Keen IO
 {% endanchor h3 %}
 
-{% anchor h4 %}
-Step 1: Create a Keen IO account
-{% endanchor h4 %}
+**Step 1: Create a Keen IO and SendGrid account**
 
 * Create a <a href="https://keen.io/signup">Keen IO account</a> (if you don't already have one)
+* Create a <a href="https://sendgrid.com/user/signup"> SendGrid account</a>
 
-{% anchor h4 %}
-Step 2: Activate SendGrid's Event Notification App
-{% endanchor h4 %}
 
-1. First click on [Mail Settings](http://app.sendgrid.com/settings/mail_settings) within Settings in the left column. 
-2. Next, activate the Event Notification Setting.
+**Step 2: Activate SendGrid's Event Notification App**
+
+Log into your SendGrid account, click on Settings from the left menu → Select <a href="http://app.sendgrid.com/settings/mail_settings">Mail Settings</a>. Then click on “Event Notification”.
+
 ![SendGrid- Mail Settings]({{root_url}}/images/keen/sendgrid_mailsettings_eventnotification.png)
-3. Enter the Keen HTTP POST URL, shown below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
-    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
-    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
-4. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check **Select All** under the *Select Actions*, or select just the ones you'd like to send to Keen IO.
 
-That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
+**Step 3: Set the HTTP POST URL (example below) as the endpoint where SendGrid will stream all of your email data. **
 
```

Comment:
@elof I'm a little confused by this step - where do I set this URL? In the SG event webhook? In the Keen app?

Reply:
You get the URL in the Keen settings UI and you set the URL in the SG settings UI. Does that make sense?

Reply:
It does! Do you have links handy for both of those pages? I don't currently have a Keen integration up, so I'm not sure what to look for.

Reply:
There isn't an easy way to link to where to get this without a Keen account. The link is embedded in the installation section of a project and contains a write key for auth

Reply:
We just made some rad changes for readability and accuracy, @ksigler7 will this still be good to launch on Wed 9/13? Thanks!